README.md

```diff
@@ -418,6 +459,15 @@ bundle exec rake jshint
 
 You can read about JSHint [here](http://jshint.com/docs/).
 
+## ESLint
+
+```
+cd client/
+yarn eslint app
+```
+
+You can read about ESLint [here](https://eslint.org/).
+
```

Comment:
Add section for Flow:

## Flow

```
cd client
yarn flow
```

You can read about Flow [here](https://flow.org).